tests/phpunit/tests/theme.php

```diff
@@ -23,13 +23,20 @@ class Tests_Theme extends WP_UnitTestCase {
 		'twentytwentytwo',
 	);
 
+	/**
+	 * Original theme directory.
+	 *
+	 * @var string
```

Comment:
```suggestion
	 * @var string[]
```

Reply:
Thanks for catching that. Fixed up in the original commit.